### DISCUSSION TITLE
Plasma Protein Binding (fu) Definition and Application in MoBi

### DISCUSSION THREAD

**Discussion by jiangz-98** [Open-Systems-Pharmacology/Forum#1869] [Questions & Problems]
Here's the definition of Fraction Unbound (plasma) in MoBi "Spatial Structure" block: 
Min(1 / (1 + (BindingPartner = Albumin ? AlbuminOntgenyFactor : (BindingPartner = Agp ? AgpOntogenyFactor : 1) ) * (1 - fu_ref) / fu_ref) * PlasmaProteinScaleFactor; 1)

How did this equation came from? If BindingPartner isn't defined as neither Albumin and Agp and PlasmaProteinScaleFactor is 1, why the fu is still adjusted from fu_ref?

Dose this Fraction Unbound (plasma) directly applying to the Organism<-VenousBlood<-Plasma<-MOLECULE<-Concentration?

Thank you for your time!
<img width="755" alt="Fu" src="https://github.com/user-attachments/assets/68fc9eb1-b338-4370-8fe6-de94c7ba7246">

**Comment by AndreDlm** [comment]
Dear @jiangz-98, 

> How did this equation came from

This equation is used for scaling fraction unbound to populations where albumin concentration is altered and consequently also fraction unbound. It is based on the equation by McNamara & Alcorn, although differently represented. For further details on this equation see the publication by [McNamara & Alcorn](https://pubmed.ncbi.nlm.nih.gov/12049488/) or by [Dallmann et al](https://pubmed.ncbi.nlm.nih.gov/28391404/) where the same equation is used.

The `Ontogeny Factor` is based on changes in albumin concentration in the pediatric population. For albumin, an `Ontogeny Factor` is implemented in the software but not for Agp ( alpha-1-acid glycoprotein; orosomucoid). Hence, the `Ontogeny Factor` for Agp is 1, resulting in similar fraction unbound in children. If the binding partner is neither Albumin nor Agp, the `Ontogeny Factor` is also 1. 

> If BindingPartner isn't defined as neither Albumin and Agp and PlasmaProteinScaleFactor is 1, why the fu is still adjusted from fu_ref?

`fu_ref` is the fraction unbound value you manually define in PK-Sim. Thus, if the `Ontogeny Factor` is 1, the equation above will yield the same value, i.e., fraction unbound = `fu_ref`.

> Dose this Fraction Unbound (plasma) directly applying to the Organism<-VenousBlood<-Plasma<-MOLECULE<-Concentration?

Yes, the Fraction Unbound (plasma) calculated from the equation above will be used in the model, e.g., to convert total Venous Blood Plasma Concentration to Unbound Concentration (this conversion is defined in the Observer Building Block).
